package/MDAnalysis/core/groups.py

```diff
@@ -2554,19 +2554,46 @@ def write(self, filename=None, file_format="PDB",
             ``"conect"``: write only the CONECT records defined in the original
             file. ``"all"``: write out all bonds, both the original defined and
             those guessed by MDAnalysis. ``None``: do not write out bonds.
-            Default os ``"conect"``.
+            Default is ``"conect"``.
+        frames:
```

Comment:
Can you add a special value "all" to be equivalent to `slice(None, None, None)`, the latter being a bit non-intuitive when someone wants to write the whole trajectory.

I see @richardjgowers 's point to make the `AtomGroup` the central point of contact for users, including writing trajectories instead of the suggestion in #1037 to use `u.trajectory.write(..., atoms=AtomGroup)`. But it should be clear how one can accomplish the "simple" task of "write a trajectory of all atoms to this file", e.g., trajectory conversion dcd -> xtc
```python
u = Universe(PSF, DCD)
u.atoms.write("new.xtc", frames="all")
```

Reply:
See also comment below on the values for `frames` for an alternative to `frames="all"`.

Perhaps
* `True`: write the *whole* trajectory if the format allows, otherwise fail with `ValueError`
* `False`: only write the current frame
* `None`: choose depending on the file format: all MultiFrameWriters (excluding PDB): choose `True`, all single frame readers (and PDB): choose `False`

Reply:
I went for `frame='all'`